README.md

```diff
@@ -1 +1,363 @@
+- permissions api: https://rems.gdi.nbis.se/api/permissions/<lifescience_user_id>
+- user: <ROBOT_USER_NAME>
+- user api key: <taken_from_REMS_database>
+```
+* The `lifescience_user_id` is not a specific value. It will be used from the LS AAI when requesting permissions.
+* The user API key is created in the previous step. In order to extract it from the database, run 
+```bash
+docker exec -it rems_db psql -U rems -h localhost -c 'select * from api_key' rems
+```
+and get the key created for the robot account.
+
+
+If everything worked well, you should be able to see the visas created by REMS in the token provided by the LS AAI for a specific user.
+
+### Using REMS
+In order to use REMS and be able to apply for access, you need to follow the steps described [here](https://github.com/GenomicDataInfrastructure/starter-kit-rems#create-your-own-data). This guide assumes that there exists an organisation, workflow etc.
+
+## Deploying Beacon2 (WIP)
+
+TODO: This part is a work in progress and should probably be skipped for now
+
+Clone the starter-kit-beacon2-ri-api repo using the following command:
+```bash
+git clone https://github.com/GenomicDataInfrastructure/starter-kit-beacon2-ri-api.git
+```
+
+Deployment instructions are available in the repo, at `https://github.com/GenomicDataInfrastructure/starter-kit-beacon2-ri-api/blob/master/deploy/README.md` 
+
+There are some port collisions with storage-and-interfaces in this repo:
+```none
+9000 is used for beacon_training_ui
+8081 is used for mongo-express
+8000 is used for keycloak
+```
+In our case, changed these are:
+```none
+9000 → 9090
+8081 → 8082
+8000 → 8083
+```
+
+Start the deployment with: 
+```bash
+docker compose up --build -d
+```
+
+Follow the instructions in the repository for loading the data, summarized:
+```bash
+for collection in datasets analyses biosamples cohorts genomiVariations individuals runs
+do
+    docker cp "./data/$collection.json" "deploy-db-1:/tmp/$collection.json"
+    
+    docker exec deploy-db-1 mongoimport --jsonArray \
+        --uri 'mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin' \
+        --file "/tmp/$collection.json" --collection "$collection"
+done
+```
+
+And finally
+```bash
+docker exec beacon python beacon/reindex.py
+```
+To create indexes for the database.
+
+You then need to set up ontologies correctly with:
+```bash
+docker exec beacon python beacon/db/extract_filtering_terms.py
+docker exec beacon python beacon/db/get_descendants.py
+```
+
+## Create endpoint for public crypt4gh key
+Create a new bucket and upload the key to that bucket. Then change the key permissions to public:
+```bash
+s3cmd -c <s3config> mb s3://gdi-public
+s3cmd -c <s3config> put c4gh.pub.pem s3://gdi-public/key/
+s3cmd -c <s3config> setacl s3://gdi-public/key/c4gh.pub.pem --acl-public
+```
+
+If you are using the Swedish GDI implementation, the public key for the next step can be found under `https://s3.sto3.safedc.net/gdi-public/key/c4gh.pub.pem`
```

Comment:
```suggestion
If you are using the Swedish GDI implementation, the public key for the previous step can be found under `https://s3.sto3.safedc.net/gdi-public/key/c4gh.pub.pem`
```

Reply:
Usage of the key `c4gh.pub.pem` appears before this sentence, it is a bit weird to mention it after that and with `next step`.

Reply:
This is actually used in the next step. The previous is only a sample for uploading the key, but doesn't have to be this name or key

Reply:
A bit confusing, maybe use <PUB-KEY> instead of `c4gh.pub.pem` in the s3cmd commands?

Reply:
Not sure I understand? Use what instead of `c4gh.pub.pem`

Reply:
sorry, edited. Markdown hide text inside `<>`